### ISSUE TITLE
Can't get it to run

### ISSUE THREAD

**Issue by alexgleason** [rusher81572/cloudExplorer#60]
Hi, I've tried using `java -jar CloudExplorer.jar` locally and it spat back a huge error.

Next, I compiled the docker image and ran it, connected via VNC, and it has an error there, too. It is very different from the other one.

![screenshot from 2018-04-23 17 51 29](https://user-images.githubusercontent.com/3639540/39155188-fb47fe96-471e-11e8-937f-a0b2d01585e4.png)

Any ideas? Thanks!

P.S. I think pushing a working Docker image to Docker Hub would be useful so we can just pull it and it will be guaranteed to work.

**Comment by rusher81572** [comment]
Thanks for the feedback. I will work on updating the Docker image. When you run it locally, what version of Java are you using?

**Comment by alexgleason** [comment]
Thanks. :smile: 

```
alex@Starman:~/Downloads/cloudExplorer$ java -version
openjdk version "9-internal"
OpenJDK Runtime Environment (build 9-internal+0-2016-04-14-195246.buildd.src)
OpenJDK 64-Bit Server VM (build 9-internal+0-2016-04-14-195246.buildd.src, mixed mode)
```

**Comment by rusher81572** [comment]
Does it work locally with the dev build (same as production but for JDK 9/10)

https://cloudexplorer.s3.amazonaws.com/cloudExplorer-dev.zip

**Comment by alexgleason** [comment]
Unfortunately, no:

```
alex@Starman:~/Downloads/cloudExplorer (2)$ java -jar CloudExplorer.jar 
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f0674f0d009, pid=15831, tid=15832
#
# JRE version: OpenJDK Runtime Environment (9.0) (build 9-internal+0-2016-04-14-195246.buildd.src)
# Java VM: OpenJDK 64-Bit Server VM (9-internal+0-2016-04-14-195246.buildd.src, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [libjava.so+0x1d009]  JNU_GetEnv+0x19
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P" (or dumping to /home/alex/Downloads/cloudExplorer (2)/core.15831)
#
# An error report file with more information is saved as:
# /home/alex/Downloads/cloudExplorer (2)/hs_err_pid15831.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Aborted (core dumped)
```

Here is the full log this time: [hs_err_pid15831.log](https://github.com/rusher81572/cloudExplorer/files/1940179/hs_err_pid15831.log)

**Comment by rusher81572** [comment]
Thank you for reporting this. This looks like a regression with the OpenJDK that Ubuntu ships with and was fixed in openjdk 9.0.4. The CloudExplorer CLI works fine but the problem occurs when trying to launch the GUI. I modified the Dockerfile and README.md and pushed a new image to Docker Hub: rusher81572/cloudexplorer. The upload to Docker Hub should be finished in about 15 min.

**Comment by rusher81572** [comment]
Closing issue. If the problem persists after my previous comment, we can reopen.
